cmd/interop/src/mls_client_impl.cpp

```diff
@@ -1062,14 +1062,16 @@ MLSClientImpl::commit(CachedState& entry,
     by_value.emplace_back(std::move(proposal));
   }
 
-  auto force_path = request->force_path();
-  auto inline_tree = !request->external_tree();
+  const auto force_path = request->force_path();
+  const auto inline_tree = !request->external_tree();
+  const auto include_membership_proof = false;
```

Comment:
Looking at this, we haven't updated this code since we moved to AnnotatedWelcome.  I'll update and request re-review.